gramps/gen/filters/rules/place/_withinarea.py

```
@@ -78,6 +78,12 @@ def prepare(self, db, user):
             self.latitude, self.longitude = conv_lat_lon(latitude,
                                                          longitude,
                                                          "D.D8")
+            if self.latitude == None or self.longitude == None:
+                print(_("ERROR: Cannot use the filter 'within area': "
```

Raise a `FilterError` here rather than using a print statement.